src/client/opamTreeCommand.ml

```diff
@@ -0,0 +1,235 @@
+(**************************************************************************)
+(*                                                                        *)
+(*    Copyright 2012-2019 OCamlPro                                        *)
+(*    Copyright 2012 INRIA                                                *)
+(*                                                                        *)
+(*  All rights reserved. This file is distributed under the terms of the  *)
+(*  GNU Lesser General Public License version 2.1, with the special       *)
+(*  exception on linking described in the file LICENSE.                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+open OpamStateTypes
+open OpamStd.Op
+
+let log fmt = OpamConsole.log "TREE" fmt
+let slog = OpamConsole.slog
+
+type package = OpamPackage.t
```

Comment:
Already defined: `OpamTypes.package`, you can open `OpamTypes` at the beginning to those types and shortcuts